tests/storage/nfs/test_nfs_sr.py

```diff
@@ -55,6 +59,29 @@ def test_snapshot(self, dispatch_nfs):
         finally:
             vm.shutdown(verify=True)
 
+    @pytest.mark.small_vm
+    @pytest.mark.parametrize('dispatch_nfs', ['vdi_on_nfs_sr', 'vdi_on_nfs4_sr'], indirect=True)
+    @pytest.mark.parametrize("vdi_op", ["snapshot", "clone"])
+    def test_coalesce(self, storage_test_vm: VM, dispatch_nfs: VDI, vdi_op):
+        vm = storage_test_vm
+        vdi = dispatch_nfs
+        vm.connect_vdi(vdi, 'xvdb')
```

Comment:
Why a hardcoded `xvdb`, when we have `*disks` fixtures?

Reply:
yes, those ones (and the 4k variant would likely allow to test coalesce on `largeblocks` too?)